packages/salesforcedx-vscode-core/package.json

```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "@salesforce/core": "2.4.1",
```

Comment:
You need to update to `@salesforce/core` version 2.5.1 here and on the rest of the modules that use it